resources/trigger_aws_test_run.py

```diff
@@ -83,6 +86,25 @@ def aws_job():
 
     get_test_run_artifacts(RUN_NAME, test_run_arn)
 
+def update_credentials():
+    with open(CUSTOM_SPECS_NAME, 'r') as stream:
```

Comment:
Could you add a docstring here?

Reply:
It feels odd to me that this uses a global variable and injects commands into CUSTOM_SPECS_NAME rather than accepting the yaml block as an input and returning it, but maybe AWS expects this to be a side effect?

Reply:
Basically, the AWS accepts the file input, instead of Setting up the values through the AWS instance.
**ref**: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/devicefarm.html#DeviceFarm.Client.schedule_run